content/en/releases/platforms.md

```diff
@@ -0,0 +1,359 @@
+| cri-tools**\*[cri-tools-version]\*\*\***[arch]\*\*.deb    | nightly/bionic<br />nightly/jessie<br />nightly/sid<br />nightly/stretch<br />nightly/trusty<br />nightly/xenial<br /><br />stable/bionic<br />stable/jessie<br />stable/sid<br />stable/stretch<br />stable/trusty<br />stable/xenial<br /><br />unstable/bionic<br />unstable/jessie<br />unstable/sid<br />unstable/stretch<br />unstable/trusty<br />unstable/xenial | nightly/bionic<br />nightly/jessie<br />nightly/sid<br />nightly/stretch<br />nightly/trusty<br />nightly/xenial<br /><br />stable/bionic<br />stable/jessie<br />stable/sid<br />stable/stretch<br />stable/trusty<br />stable/xenial<br /><br />unstable/bionic<br />unstable/jessie<br />unstable/sid<br />unstable/stretch<br />unstable/trusty<br />unstable/xenial | nightly/bionic<br />nightly/jessie<br />nightly/sid<br />nightly/stretch<br />nightly/trusty<br />nightly/xenial<br /><br />stable/bionic<br />stable/jessie<br />stable/sid<br />stable/stretch<br />stable/trusty<br />stable/xenial<br /><br />unstable/bionic<br />unstable/jessie<br />unstable/sid<br />unstable/stretch<br />unstable/trusty<br />unstable/xenial | nightly/bionic<br />nightly/jessie<br />nightly/sid<br />nightly/stretch<br />nightly/trusty<br />nightly/xenial<br /><br />stable/bionic<br />stable/jessie<br />stable/sid<br />stable/stretch<br />stable/trusty<br />stable/xenial<br /><br />unstable/bionic<br />unstable/jessie<br />unstable/sid<br />unstable/stretch<br />unstable/trusty<br />unstable/xenial |
```

Comment:
Seems [the format with `***`](https://deploy-preview-48457--kubernetes-io-main-staging.netlify.app/releases/platforms/#debs) not correct

Reply:
I think we can just link to pkg.k8s.io here, done that.